test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/DefaultEditorTemplatesTest.cs

```diff
@@ -822,16 +824,17 @@ public void Editor_InputTypeDateTime_RendersAsDateTime()
         // Html5DateRenderingMode.Rfc3339 is enabled by default.
         [Theory]
         [InlineData(null, null, "2000-01-02T03:04:05.060-05:00", "text")]
-        [InlineData("date", "{0:d}", "2000-01-02", "date")]
```

Comment:
Why remove this data? Same question for the `Editor_FindsCorrectDateOrTimeTemplate_WithTimeOffset_NotRfc3339(...)` data.

Reply:
Note `Editor_FindsCorrectDateOrTimeTemplate_WithTimeOffset(...)` and `Editor_FindsCorrectDateOrTimeTemplate_ForDateTime(...)` use different `Model` types (`DateTimeOffset` versus `DateTime`) and those types are handled differently.